src/permissions/permissions.service.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Injectable } from '@nestjs/common';
+import { User } from '../users/user.entity';
+import { Note } from '../notes/note.entity';
+
+export enum GuestPermission {
```

Comment:
Shouldn't this be in a separate file?

Reply:
Yes but guest permissions have to be implemented in the config anyway.

Reply:
If you intend to move this to the config module at a later time, please add a todo.